WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -914,6 +922,16 @@ private function validate_prefixes() {
 				continue;
 			}
 
+			if ( mb_strlen( $prefix ) < self::MIN_PREFIX_LENGTH ) {
+				$this->phpcsFile->addError(
+					'The "%s" prefix is too short. Short prefixes are not unique enough and may still cause name collisions with other plugins.',
```

Comment:
Prefixes is not just about plugins, but also about themes and about colliding with WP itself or even PHP.

Can we make this message a little more generic ?